Source/IC.cpp

```diff
@@ -26,4 +55,136 @@ TaylorGreen::TaylorGreen()
     }
 }
 
+ViscosityTest::ViscosityTest()
+{
+    amrex::ParmParse pp(identifier());
+    pp.query("density", m_op.density);
+
+    amrex::Vector<amrex::Real> velocity{AMREX_D_DECL(0, 0, 0)};
+    pp.queryarr("velocity", velocity, 0, AMREX_SPACEDIM);
+    for (int n = 0; n < velocity.size(); n++) {
+        m_op.velocity[n] = velocity[n];
+    }
+
+    pp.query("waveLength", m_op.waveLength); // ns:wavelength of the
+                                             // perturbation
+    std::string m_model_type;
+    pp.query("model_type", m_model_type); // ns: default is "isothermal".
+                                          // "energyD3Q27" activates product
+                                          // equilibrium, energy equation etc.
+
+    if (m_model_type == "energyD3Q27") {
```

Comment:
Question for next we meeting: should we just deprecate the other model and always use the energyD3Q27 model? that way we don't have to support both?